frontend/src/components/nodes/Nodes.tsx

```diff
@@ -2,60 +2,62 @@ import React from "react";
 
 import NodesApi, * as N from "../../libraries/explorer-wamp/nodes";
 
-import autoRefreshHandler from "../utils/autoRefreshHandler";
-import FlipMove from "../utils/FlipMove";
-
 import NodeRow from "./NodeRow";
+import ValidatorRow from "./ValidatorRow";
+import PaginationSpinner from "../utils/PaginationSpinner";
 
-import { OuterProps } from "../accounts/Accounts";
-
-interface Props extends OuterProps {
+interface Props {
   role: string;
 }
-export default class extends React.Component<Props> {
-  static defaultProps = {
-    count: 15,
+
+interface State {
+  onlineNodes?: N.NodeInfo[];
+  validators?: any;
+}
+export default class extends React.Component<Props, State> {
+  state: State = {};
+
+  getNonValidatingNodes = async () => {
+    return await new NodesApi().getOnlineNodes();
   };
 
-  fetchNodes = async (count: number, paginationIndexer?: string) => {
-    return await new NodesApi().getNodes(
-      count,
-      this.props.role,
-      paginationIndexer
-    );
+  getNodes = async () => {
+    if (this.props.role === "validators") {
+      let validators = await new NodesApi().queryValidators();
+      this.setState({ validators });
+    }
+    if (this.props.role === "online-nodes") {
+      let onlineNodes = await this.getNonValidatingNodes();
+      this.setState({ onlineNodes });
+    }
   };
 
   componentDidUpdate(prevProps: any) {
     if (this.props.role !== prevProps.role) {
-      this.autoRefreshNodes = autoRefreshHandler(Nodes, this.config);
+      this.getNodes();
```

Comment:
I remember we discuss this before and the answer is delete the auto refresh somehow. However, if you re think we need auto refresh, here are two methods:
1. use pub  and lib since I actually make this in the PR 
2. write the auto refresh logic in this component

Reply:
I find it confusing to see `last seen X minutes ago` if I keep the tab open for a while, so this needs to be resolved one way or another. I leave the implementation detail decision on you. Let me know if you need to brainstorm it (keep in mind that some nodes may go offline and back online, and new ones can get online after an initial page load, etc).

Reply:
@icerove this really needs to be addressed ^^^. Let me know if you want to do this in a separate PR or whatnot.

Reply:
in a seperate PR